tests/acquisition/covid_hosp/state_daily/test_network.py

```diff
@@ -24,3 +32,17 @@ def test_fetch_metadata(self):
 
       self.assertEqual(result, sentinel.json)
       func.assert_called_once_with(dataset_id=Network.DATASET_ID)
+
+  def test_fetch_revisions(self):
+    """Scrape CSV files from revision pages"""
+
+    with patch.object(requests, 'get',
+                      side_effect=list(self.test_utils.load_sample_revisions())) as requests_get:
+      result = Network.fetch_revisions(20201210)
+
+      self.assertEqual(requests_get.call_count, 4)
+      self.assertEqual(result, [
+        "https://healthdata.gov/sites/default/files/reported_hospital_utilization_20210130_2344.csv",
+        "https://healthdata.gov/sites/default/files/reported_hospital_utilization_20210129_1606.csv",
+        "https://healthdata.gov/sites/default/files/reported_hospital_utilization_20210128_2205.csv"
```

Comment:
why are there only 3 results here?

Reply:
The fourth URL is before the "last issue" and thus should be excluded from this batch.

I'll add a comment to clarify.